### PR TITLE
adjusted dataset format to append all scores in one file. Evaluating …

### PR DESCRIPTION
**Pull request by pedrofbo**
…program was commented and optimized to fit the new dataset format. 'ponto_virada' script was made, but still needs adjustments

### CODE REVIEW

File: chess engine/evaluate_games.py [hunk 1]
```diff
@@ -6,54 +6,95 @@
...
-        print(f'Arquivo pgn {i} não existe')
\ No newline at end of file
+        print(f'Arquivo pgn {i} não existe')
+
+scores_file.close()
+
+#salva o id do último movimento analisado
+last_id = open("last_id.txt", "w")
+last_id.write(f'{move_id}')
+last_id.close()
```
Comment (douglasramos) [inline chess engine/evaluate_games.py:100]: add a new line to the end of file

File: chess engine/ponto_virada.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    for i in range(1, len(scores)):
+        if type(scores[i]) == int and type(scores[i-1]) == int:
+            if abs(scores[i]) >= (abs(scores[i - 1]) + 150) and abs(scores[i - 1]) < 150:
+                pontos_virada.append(1)
+            else:
+                pontos_virada.append(0)
+        else:
+            pontos_virada.append(0)
+    
+    return pontos_virada
```
Comment (douglasramos) [inline chess engine/ponto_virada.py:14]: same here

File: chess engine/ponto_virada.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+def pontosVirada (scores):
+    pontos_virada = [0]
+
+    for i in range(1, len(scores)):
+        if type(scores[i]) == int and type(scores[i-1]) == int:
+            if abs(scores[i]) >= (abs(scores[i - 1]) + 150) and abs(scores[i - 1]) < 150:
+                pontos_virada.append(1)
+            else:
+                pontos_virada.append(0)
+        else:
```
Comment (douglasramos) [inline chess engine/ponto_virada.py:11]: this condition should be possible?

File: chess engine/scores.csv [hunk 4]
```diff
@@ -0,0 +1,376 @@
+Id, Pgn Number, Event, Move Number, Move, Score, Comment
```
Comment (douglasramos) [inline chess engine/scores.csv:1]: I'm not sure if we should check this file into the repository. We could upload it to the drive and keep in here only the source code

**Review by douglasramos — CHANGES_REQUESTED** [review]

File: chess engine/evaluate_games.py (hunk 1 above)
Comment (pedrofbo) [reply chess engine/evaluate_games.py:100]: check

File: chess engine/ponto_virada.py (hunk 2 above)
Comment (pedrofbo) [reply chess engine/ponto_virada.py:14]: check

File: chess engine/scores.csv (hunk 4 above)
Comment (pedrofbo) [reply chess engine/scores.csv]: file deleted accordingly

**Review by douglasramos — APPROVED** [review]

File: chess engine/ponto_virada.py (hunk 3 above)
Comment (pedrofbo) [reply chess engine/ponto_virada.py:13]: made a comment on the code to address this

**Pull request merged**